ibm/mas_devops/common_tasks/get_ingress_cert.yml

```diff
@@ -1,12 +1,16 @@
 ---
 # 1. Lookup for router-certs-default secret
 # -----------------------------------------------------------------------------
+- name: set ocp ingress tls secret
+  set_fact:
```

Comment:
@racree thinking again here... I would add a `when` clause here to only to this set_fact when `ocp_ingress_tls_secret` is not defined. I am thinking that this var must be set in a playbook or in a role calling such common task (see documentation comment within this same review)...